CHANGELOG.md

```diff
@@ -1,13 +1,52 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-06-27
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled as v20240627.
+
+### Announcements
+
+* Starting 1.30 Kubernetes version and 1.27 LTS versions, beta APIs will be disabled by default, when you upgrade to [them](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#before-you-begin). There will be an option provided to explicitly enable beta APIs closer to the 1.30 release.
+* 1.30 is the next LTS version after 1.27. Customers will have at least 6 months of migration time available between 1.27 LTS to 1.30 LTS i.e before the 1.27 LTS goes End of Life by July 2025. Refer [long term support](https://learn.microsoft.com/azure/aks/long-term-support) for more information.
+
+### Release Notes
+
+* Features:
+  * AKS patch versions 1.29.5, 1.28.10, 1.27.14 are now available. Refer to [version support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#kubernetes-version-support-policy) and [upgrading a cluster](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli) for more information.
```

Comment:
did we deprecate any patch.. if you see anything in the PR list pls call it out too.

Reply:
I see 1.27.9, 1.28.5, and 1.29.2 in the red. @kaarthis can you follow up with the team and confirm that these are being removed? and I'll add it here